src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -344,4 +350,18 @@
   <ItemGroup>
     <DesignTimeTargetsFile Include="ProjectSystem\DesignTimeTargets\*.targets" />
   </ItemGroup>
+  <ItemGroup>
+    <None Update="ProjectSystem\Rules\ApplicationDefinition - Copy.xaml">
```

Comment:
This doesn't look right :)

Reply:
Ugh.. I specifically did the file duplication out of VS to stop it doing this, must have switched back too quickly.